src/windows/win-dumpconfigurator.ps1

```diff
@@ -0,0 +1,70 @@
+ . .\src\windows\common\setup\init.ps1
+<#
+.SYNOPSIS
+    This script will change the dump configuration without requiring a reboot.
+
+    Prerequisites (Links below):     
+    Kdbgctrl.exe (From the Debugging Tools for Windows.)
+ 
+ 
+.NOTES
+    Name: win-dumpconfigurator.ps1
+    Author: Microsoft CSS
+    Version: 1.1
+    Created: 2021-Mar-1
+ 
+ 
+.EXAMPLE
+    ./win-dumpconfigurator.ps1 -DumpType full -DumpFile C:\Dumps\Memory.dmp -DedicatedDumpFile D:\dd.sys
+ 
+ 
+.LINK
+    Debugging Tools for Windows -- https://docs.microsoft.com/en-us/windows-insider/flight-hub/
+#>
+Param(
+[parameter()] [switch]$OneDump,
+[parameter()] [ValidateSet("active", "automatic", "full", "kernel", "mini" )] [string]$DumpType,
+[parameter()] [string]$DumpFile,
+[parameter()] [string]$DedicatedDumpFile="C:\DedicatedDumpFile.sys",
+[parameter()] [string]$VMName,
+[parameter()] [string]$ResourceGroup)
+
+#For testing purposes
+#$DumpType = 'full'
+#$DedicatedDumpFile="C:\DedicatedDumpFile.sys"
```

Comment:
Lets remove these when we actually merge the changes in

Reply:
Understood, I've removed them.